response_operations_ui/static/css/override.css

```diff
@@ -29,9 +29,11 @@ footer{
;opacity:0}.btn .btn__icon--print-link{width:1em;background-image:url(../img/icons/icons--print-focus.svg)}.btn .btn__icon--print-link:before{background-image:url(../img/icons/icons--print-hover.svg)}.btn .btn__icon--print-link:after{background-image:url(../img/icons/icons--print-focus.svg)}.btn--menu,.btn--secondary{background-color:#e4e8eb;color:#4263c2;border:2px solid #e4e8eb;padding:13.5px 18px;padding:.75rem 1rem;margin:0 0 18px;margin:0 0 1rem;font-size:18px;font-size:1rem;font-weight:600;border-radius:3px;display:inline-block;text-rendering:optimizeLegibility;text-decoration:none;width:100%;-webkit-transition:background-color .2s ease-in,color .2s ease-in,border .2s ease-in;transition:background-color .2s ease-in,color .2s ease-in,border .2s ease-in;*display:inline;*zoom:1}.btn--menu:focus,.btn--menu:hover,.btn--secondary:focus,.btn--secondary:hover{text-decoration:none;color:#033e58;background-color:#c7cfd5;border-color:#c7cfd5}.btn--menu:focus,.btn--secondary:focus{outline-offset:3px;outline:3px solid #fe781f}@media only screen and (min-width:500px){.btn--menu,.btn--secondary{width:auto}}.btn--menu:focus,.btn--menu:hover,.btn--secondary:focus,.btn--secondary:hover{background-color:#d5dce0;border-color:#d5dce0}.btn--ghost{background-color:transparent;border:2px solid transparent;padding:13.5px 18px;padding:.75rem 1rem;margin:0 0 18px;margin:0 0 1rem;font-size:18px;font-size:1rem;font-weight:600;border-radius:3px;display:inline-block;text-rendering:optimizeLegibility;text-decoration:none;width:100%;-webkit-transition:background-color .2s ease-in,color .2s ease-in,border .2s ease-in;transition:background-color .2s ease-in,color .2s ease-in,border .2s ease-in;color:#fff;border:2px solid hsla(0,0%,100%,.6);padding:9px 18px;padding:.5rem 1rem;*display:inline;*zoom:1}.btn--ghost:focus,.btn--ghost:hover{text-decoration:none;color:#fff;border-color:transparent}.btn--ghost:focus{outline-offset:3px;outline:3px solid #fe781f}@media only screen and (min-width:500px){.btn--ghost{width:auto}}.btn--ghost:focus,.btn--ghost:hover{background-color:transparent;border-color:#fff}.btn--menu,.btn--small{padding:7.2px 9px;padding:.4rem .5rem;font-weight:600;font-size:16.2px;font-size:.9rem}.btn--loader{-webkit-transition:color .3s ease-out;transition:color .3s ease-out;position:relative}.btn--loader:after{display:block;content:"";width:36px;width:2rem;height:36px;height:2rem;opacity:0;position:absolute;top:0;right:0;bottom:0;left:0;margin:auto;background-image:url(../img/loader.svg);background-repeat:no-repeat;background-position:50%;background-size:1.5rem;-webkit-transition:opacity .3s ease-out .2s;transition:opacity .3s ease-out .2s;pointer-events:none}.btn--loader.is-loading{color:transparent}.btn--loader.is-loading:after{opacity:1}.btn-group .btn-group__btn{display:inline-block;margin-right:18px;margin-right:1rem;*display:inline;*zoom:1}.btn-group .btn-group__btn:last-of-type{margin-right:0}@media only screen and (min-width:740px){.btn--menu{display:none}}
```

Comment:
I think we should expand this. It will be really difficult to maintain any styling changing if we keep the source minified